lib/tasks/active_record_views.rake

```diff
@@ -52,12 +52,46 @@ Rake::Task[schema_rake_task].enhance do
     pg_tasks = tasks.send(:class_for_adapter, adapter).new(config)
     pg_tasks.send(:set_psql_env)
 
-    require 'shellwords'
-    system("pg_dump --data-only --table=active_record_views #{Shellwords.escape database} >> #{Shellwords.escape filename}")
-    raise 'active_record_views metadata dump failed' unless $?.success?
+    begin
+      active_record_views_dump = Tempfile.open("active_record_views_dump.sql")
+      require 'shellwords'
+      system("pg_dump --data-only --no-owner --table=active_record_views #{Shellwords.escape database} >> #{Shellwords.escape active_record_views_dump.path}")
+      raise 'active_record_views metadata dump failed' unless $?.success?
 
-    File.open filename, 'a' do |io|
-      io.puts 'UPDATE public.active_record_views SET refreshed_at = NULL WHERE refreshed_at IS NOT NULL;'
+      ActiveRecordViews::RakeTaskUtilities.remove_sql_header_comments(active_record_views_dump.path)
+
+      File.open filename, 'a' do |io|
+        File.foreach(active_record_views_dump.path) do |line|
+          io.puts line
+        end
+
+        io.puts 'UPDATE public.active_record_views SET refreshed_at = NULL WHERE refreshed_at IS NOT NULL;'
+      end
+    ensure
+      active_record_views_dump.unlink
```

Comment:
I know it doesn’t matter since the rake task is running an ephemeral process, but I feel like we should `close` and `unlink` here as a general good practice.

Reply:
Done!